README.md

```diff
@@ -74,22 +74,43 @@ java -jar build/libs/Fruzhin-0.1.0.jar -n polkadot --node-mode full --sync-mode
 - `--sync-mode` could be `full` or `warp`
 
 ### Local development
+In order to use the Fruzhin node for local development you will first need to start another node that would serve as a
+peer. 
 
-1. Run a local Polkadot node with ```polkadot --dev``` command. (The default starting port is 9944)
-2. Fetch the chain spec
+For the sake of this example we will use [Paritytech's implementation](https://github.com/paritytech/polkadot-sdk).
+If you are not familiar with how to run a node see [this](https://wiki.polkadot.network/docs/maintain-sync#setup-instructions).
+
+Once you have successfully built the Polkadot project run the node via ``polkadot --dev``.
+(The node starts on port 9944 by default)
+
+Now you have 2 options:
+- Use the automated `local_dev.sh` script
+- Manual setup.
+
+#### Automated script
+1. Install [JQ](https://github.com/jqlang/jq).
+
+   `sudo apt-get install jq` Ubuntu
+   
+   `brew install jq` MacOS
+
+2. Head to the main directory of Fruzhin and run `chmod +x script.sh`
+3. Execute the script `./local_dev.sh`
```

Comment:
It should be ./chmod +x local_dev.sh
but whatsoever it's commited to the git repository with the executable flag already included, so this step is obsolete


```suggestion
2. Head to the main directory of Fruzhin execute the script `./local_dev.sh`
```